### PR TITLE
Move test to correct place

### PR DESCRIPTION
**Pull request by AdrianTeng**

### CODE REVIEW

**Comment by AdrianTeng** [comment]
@jamesblackburn

File: tests/integration/tickstore/test_tickstore.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from arctic.arctic import Arctic
+
+
+def test_stringify_tickstore_library(mongo_host):
```
Comment (jamesblackburn) [inline tests/integration/tickstore/test_tickstore.py]: Can we make this a unit test? Spinning up a mongodb and creating libraries, etc. is inefficient just to call `str(...)` :)

Comment (AdrianTeng) [inline tests/integration/tickstore/test_tickstore.py]: done

**Comment by jamesblackburn** [comment]
+1 thanks Adrian!

**Comment by AdrianTeng** [comment]
Someone is fiddling around with Travis, but I don't think it's ready yet. Will ignore for now

**Pull request merged**